Cargo.toml

```diff
@@ -24,16 +24,3 @@ toml = "0.7"
 [profile.dist]
 inherits = "release"
 lto = "thin"
-
-# Config for 'cargo dist'
-[workspace.metadata.dist]
-# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
-cargo-dist-version = "0.4.0"
```

Comment:
FYI the installed location of the binary likely changed with this update, if you have any places that refer to `.cargo/bin` they'll be `./local/bin`